common/src/main/scala/explore/components/state/IfLogged.scala

```diff
@@ -25,6 +26,9 @@ final case class IfLogged(view: View[RootModel])(val render: (UserVault, IO[Unit
 object IfLogged {
   type Props = IfLogged
 
+  protected implicit val propsReuse: Reusability[Props] =
+    Reusability.derive && Reusability.by(_.render)
```

Comment:
Does `Reusability.derive` only look at the the first parameter list? Or does it not work with the function reusability? Or both?

Reply:
It only looks at the first parameter list.